### ISSUE TITLE
Wan2.1-T2V-1.3B 在 github repo中推理和 diffusers版本的推理结果差距较大

### ISSUE THREAD

**Issue by sijeh** [Wan-Video/Wan2.1#178]
非常好的工作！

你好，我使用最新发布的diffusers版本，并根据 [示例](https://huggingface.co/Wan-AI/Wan2.1-T2V-1.3B-Diffusers) 推理 Wan2.1-T2V-1.3B 的模型，发现结果与这个代码仓库中推理出来的结果差距加大，麻烦能看看是什么情况吗？

### 以下是github仓库推理出来的结果

=================================
[https://www.alipan.com/s/p4EpBBFDAqr](https://www.alipan.com/s/p4EpBBFDAqr)

### 以下是diffusers推理出来的结果
=================================
[https://www.alipan.com/s/v4tmBjmQL69](https://www.alipan.com/s/v4tmBjmQL69)

### 以下是我使用 diffusers版本推理的code以便复现
=================================
```
import torch
import os
import os.path as osp
from diffusers import AutoencoderKLWan, WanPipeline
from diffusers.utils import export_to_video

# Available models: Wan-AI/Wan2.1-T2V-14B-Diffusers, Wan-AI/Wan2.1-T2V-1.3B-Diffusers
model_id = "Wan2.1-T2V-1.3B-Diffusers"
vae = AutoencoderKLWan.from_pretrained(model_id, subfolder="vae", torch_dtype=torch.float32)
pipe = WanPipeline.from_pretrained(model_id, vae=vae, torch_dtype=torch.bfloat16)

save_dir = 'samples/'
os.makedirs(save_dir, exist_ok=True)
pipe.to("cuda")

# prompt = "A cat walks on the grass, realistic"

txt2vid_prompt=[
    'a penguin is seen standing on a yellow surfboard, riding a wave in the ocean. The penguin appears to be balancing skillfully on the surfboard, with waves crashing around it, creating a dynamic and adventurous scene.',
    'A train is moving on the track. The scene is a train station with overhead electrical lines and structural elements visible. The weather seems overcast, adding a subdued atmosphere to the setting.',
    'A person wearing headphones and carrying a backpack walks through a sunlit cobblestone square in a historic European city. The scene is bathed in warm light, creating a serene and picturesque atmosphere with classic architecture in the background.',
    'Two adorable golden retriever puppies are sitting on a grassy field adorned with scattered orange flowers. Their tongues out, they appear joyful and playful, adding a heartwarming touch to the serene outdoor setting.',
    'A joyful Corgi with a pink harness is walking along a paved path beside a body of water, likely a lake or pond. The dog appears happy and energetic, with its tongue out, suggesting it might be panting or excited. The background includes some greenery and blurred figures, indicating a park-like setting where people are enjoying outdoor activities.',
    "A motorcyclist in full gear leans into a sharp turn on a racetrack, showcasing impressive riding skills and the dynamic motion of the motorcycle. The rider's helmet and protective suit, along with the colorful design of the motorcycle, highlight the intensity and excitement of the race.",
    "A wild pig swims in the clear turquoise waters of a tropical sea, with a seagull perched casually on its back. The idyllic scene is set against a backdrop of a sandy shoreline and lush greenery, creating a picturesque and unusual interaction between land and sea creatures. "
]

# negative_prompt = "Bright tones, overexposed, static, blurred details, subtitles, style, works, paintings, images, static, overall gray, worst quality, low quality, JPEG compression residue, ugly, incomplete, extra fingers, poorly drawn hands, poorly drawn faces, deformed, disfigured, misshapen limbs, fused fingers, still picture, messy background, three legs, many people in the background, walking backwards"
negative_prompt = ""

for i, prompt in enumerate(txt2vid_prompt):
    save_path = osp.join(save_dir, f'example_{i}.mp4')
    if osp.exists(save_path):
        continue

    output = pipe(
        prompt=prompt,
        negative_prompt=negative_prompt,
        height=480,
        width=832,
        num_frames=81,
        guidance_scale=5.0,
        num_inference_steps=25,
    ).frames[0]
    export_to_video(output, save_path, fps=15)
```

期待您的回复！

**Comment by wan-x-ai** [comment]
Sorry, the code examples on diffusers haven't been updated yet. Please try the diffusers code examples in our GitHub repo instead.

**Comment by lith0613** [comment]
> Sorry, the code examples on diffusers haven't been updated yet. Please try the diffusers code examples in our GitHub repo instead.

您好，首先感谢你们这项非常棒的工作。发现类似的情况，就是diffusers版本如果直接复用pipeline里面的schedule推理结果就很差，但是按照github的diffusers例子，加上schedule就好了：
`flow_shift = 5.0 # 5.0 for 720P, 3.0 for 480P
scheduler = UniPCMultistepScheduler(prediction_type='flow_prediction', use_flow_sigmas=True, num_train_timesteps=1000, flow_shift=flow_shift)
pipe = WanPipeline.from_pretrained(model_id, vae=vae, torch_dtype=torch.bfloat16)
pipe.scheduler = scheduler`
方便解释下为何不能直接用默认的schedule吗

**Comment by wan-x-ai** [comment]
> > Sorry, the code examples on diffusers haven't been updated yet. Please try the diffusers code examples in our GitHub repo instead.
> 
> 您好，首先感谢你们这项非常棒的工作。发现类似的情况，就是diffusers版本如果直接复用pipeline里面的schedule推理结果就很差，但是按照github的diffusers例子，加上schedule就好了： `flow_shift = 5.0 # 5.0 for 720P, 3.0 for 480P scheduler = UniPCMultistepScheduler(prediction_type='flow_prediction', use_flow_sigmas=True, num_train_timesteps=1000, flow_shift=flow_shift) pipe = WanPipeline.from_pretrained(model_id, vae=vae, torch_dtype=torch.bfloat16) pipe.scheduler = scheduler` 方便解释下为何不能直接用默认的schedule吗

The default scheduler works, but you'll need to adjust the sampling shift parameters (5.0 for 720P, 3.0 for 480P), because the default ones aren't aligned with the training setup.

**Comment by sijeh** [comment]
我通过调整 flow_shift 解决了该问题，对于1.3b T2V， flow_shift设置为8.0时能够生成较好的结果

**Comment by cats-food** [comment]
我也发现diffusers版本和原repo版有一定差距，即使我把flow_shift=8.0还是没有原repo的效果好，所有参数包括Seed，negative prompt都是一样的，是什么原因呢？

**Comment by dejian-lc** [comment]
> 我也发现diffusers版本和原repo版有一定差距，即使我把flow_shift=8.0还是没有原repo的效果好，所有参数包括Seed，negative prompt都是一样的，是什么原因呢？

话说原始权重和diffuser版本权重有啥区别嘛

**Comment by nitinmukesh** [comment]
@cats-food  @dejian-lc

Post the issue here with code and output

https://github.com/huggingface/diffusers/issues
